example/log_viewer.rb

```diff
@@ -0,0 +1,10 @@
+require "yast"
+
+require "systemd_journal/entries_dialog"
+require "systemd_journal/query_presenter"
+
+params = Yast::WFM.Args
+raise "missing param for log entries" if params.empty?
+
+query = SystemdJournal::QueryPresenter.new(interval: "0", filters: { "unit" => params })
```

Comment:
See below. I would change the `QueryPresenter` API a bit to make possible to do something like this:

```ruby
query = SystemdJournal::Query.new(interval: "0", filters: { "unit" => params })
SystemdJournal::EntriesDialog.new(query: query).run
```

Because having to provide a presenter object instead of the real thing to the dialog feels very weird.